ledger/controllers/wallet.js

```diff
@@ -47,6 +47,9 @@ const read = function (runtime, apiVersion) {
 
     if (apiVersion === 2) {
       result = underscore.extend(result, { addresses: wallet.addresses })
+      if (runtime.registrars.persona) {
+        result = underscore.extend(result, { parameters: runtime.registrars.persona.payload || {} })
```

Comment:
shouldn't this be `properties` to match the balance response schema?

Reply:
good question. let me check!

Reply:
it should be `parameters` ... fixed! thanks for catching that.